setup.py

```diff
@@ -4,16 +4,29 @@
 import os
 import sys
 from setuptools import setup, find_packages
-from sys import platform
+from typing import Tuple
+
+
+def package_metadata() -> Tuple[str, str]:
+    sys.path.insert(0, os.path.join("graspy", "version"))  # TODO: #454 Change path in https://github.com/microsoft/graspologic/issues/454
+    from version import name, version
```

Comment:
out of curiosity, why the within-function import instead of importing into the global namespace?